migrate/live-sync.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Live sync
```

Comment:
I suggest we add Postgres as suffix as its targeted for only Postgres

Reply:
Gone the whole hog: [Livesync from Postgres to Timescale Cloud](https://docs-dev.timescale.com/docs-3669-docs-rfc-add-new-live-sync-guide-to-the-migration-section/migrate/3669-docs-rfc-add-new-live-sync-guide-to-the-migration-section/livesync/#livesync-from-postgres-to-timescale-cloud)